.env.example

```
@@ -1,2 +1,12 @@
 POLAR_API_BASE_URL=https://api.polar.sh/v1
-LANGBASE_API_KEY="your_langbase_api_key_here"
\ No newline at end of file
+LANGBASE_API_KEY="your_langbase_api_key_here"
+
+DATABASE_URL="your_database_connection_string_here"
+NEXTAUTH_URL=http://localhost:3000
+NEXTAUTH_SECRET="your_nextauth_secret_here"
+
+EMAIL_SERVER_HOST="smtp.example.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)